firebase/functions/src/index.ts

```diff
@@ -194,6 +193,38 @@ export const addUserToChain = functions
     }
   });
 
+const notifyChainAdmin = async (chainId: string, newUserId: string) => {
+  const chain = await db.collection("chains").doc(chainId).get();
+  const chainAdminUid = await chain.get("chainAdmin");
+
+  const chainAdmin = await admin.auth().getUser(chainAdminUid);
+  const adminName = chainAdmin.displayName;
+  const adminEmail = chainAdmin.email;
+
+  const newUser = await admin.auth().getUser(newUserId);
+  const name = newUser.displayName;
+  const email = newUser.email;
+  const phone = newUser.phoneNumber;
+
+  db.collection("mail").add({
+    to: adminEmail,
+    message: {
+      subject: "A participant just joined your Loop!",
+      html: ` <p>Hi, ${adminName}</p>
+                        <p>A new participant just joined your loop</p>
+                        <p>Please find below all contacts:</p>
```

Comment:
I don't think that's correct english, I'd use:

`<p>The participant's contact information:</p>`